saleor/graphql/shop/tests/test_shop.py

```diff
@@ -1417,6 +1424,10 @@ def test_order_settings_update_by_app(
     response_settings = content["data"]["orderSettingsUpdate"]["orderSettings"]
     assert response_settings["automaticallyConfirmAllNewOrders"] is False
     assert response_settings["automaticallyFulfillNonShippableGiftCard"] is False
+    assert (
+        response_settings["defaultTransactionFlowStrategy"]
+        == TransactionFlowStrategyEnum.CHARGE.name
```

Comment:
Same here

Reply:
I've added additional assert to confirm that it matches to the channel setting. Thanks!